### PR TITLE
Add translations support + Spanish translation

### PR DESCRIPTION
**Pull request by oscfdezdz**

### CODE REVIEW

File: extension.js [hunk 1]
```diff
@@ -66,7 +72,10 @@ const BatteryConservationIndicator = GObject.registerClass(
       const status = Shell.get_file_contents_utf8_sync(sys_conservation);
       const active = (status.trim() == "1");
       this._indicator.visible = active;
-      this._item.label.text = _(`Turn Conservation Mode ${active ? 'Off' : 'On'}`);
+      if (active)
+      	this._item.label.text = _("Turn Conservation Mode Off")
+      else
+      	this._item.label.text = _("Turn Conservation Mode On");
```
Comment (oscfdezdz) [inline extension.js:78]: I changed this line because in languages like Spanish the verb turn on/off is a single word verb.

**Comment by oscfdezdz** [comment]
Marking as a draft, as I've found a cleaner way to translate extensions.

**Comment by oscfdezdz** [comment]
`make build` : generates .pot and packs the extension
`make install` : installs it
`make all` : `make build` and `make install`

**Comment by laurento** [comment]
Many thanks Òscar for your contribution. I'll test and merge in the coming days.

**Comment by oscfdezdz** [comment]
> `make build` : generates .pot and packs the extension

On second thought, it's probably better to regenerate the .pot file and pack the extension in separate steps

**Pull request merged**